server/src/main/java/org/apache/iotdb/db/qp/physical/PhysicalPlan.java

```diff
@@ -436,6 +444,8 @@ public static PhysicalPlan create(ByteBuffer buffer) throws IOException, Illegal
     DROP_TRIGGER,
     START_TRIGGER,
     STOP_TRIGGER,
+    CREATE_FUNCTION,
+    DROP_FUNCTION,
```

Comment:
It's better to add the new plan to the end so that we can upgrade it  directly, otherwise the old version can not upgrade it correctly.

Reply:
Is this conversation resolved? :D